src/consensus/params.cpp

```diff
@@ -218,12 +230,29 @@ namespace Consensus {
         vFundingStreams[idx] = FundingStream::ParseFundingStream(*this, keyConstants, startHeight, endHeight, strAddresses);
     };
 
-    FundingStreamAddress FundingStream::RecipientAddress(const Consensus::Params& params, int nHeight) const
+    void Params::AddZIP207LockboxStream(
+        const KeyConstants& keyConstants,
+        FundingStreamIndex idx,
+        int startHeight,
+        int endHeight)
+    {
+        auto intervalCount = FundingPeriodIndex(startHeight, endHeight - 1);
```

Comment:
Required interval count is last required index + 1.
```suggestion
        auto intervalCount = FundingPeriodIndex(startHeight, endHeight - 1) + 1;
```